cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/passes/DynamicInvokeResolver.kt

```diff
@@ -149,7 +151,26 @@ class DynamicInvokeResolver(ctx: TranslationContext) : ComponentPass(ctx) {
                     continue
                 }
             }
-            curr.prevDFG.forEach(Consumer(work::push))
+            // Do not consider the base for member expressions, we have to know possible values of
+            // the member (e.g. field).
+            var prevDFGToPush =
```

Comment:
We should probably rather filter for data flows which have the granularity set to full or omitted (which is the same as full).

This also applies to the analysis queries I guess.